src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -133,17 +137,22 @@ private TableOfContents(PDDocument document, Bundle bundle) throws IOException {
                 addText(document, getPage(), bundle.getDescription(), 50,80, PDType1Font.HELVETICA,12);
             }
 
-            addCenterText(document, getPage(), "Contents", 130);
+            addCenterText(document, getPage(), INDEX_PAGE, 130);
+            addText(document, getPage(), PAGE, 480,165, PDType1Font.HELVETICA,12);
         }
 
-        public void addDocument(String documentTitle, int pageNumber) throws IOException {
+        public void addDocument(String documentTitle, int pageNumber, int noOfPages) throws IOException {
             final float yOffset = getVerticalOffset();
             final PDPage destination = document.getPage(pageNumber);
-            final String text = documentTitle; // + ", p" + (pageNumber + 1);
+            final String text = documentTitle;
 
             addLink(document, getPage(), destination, text, yOffset,PDType1Font.HELVETICA,12);
-            final String pageNo = ", p" + (pageNumber + 1);
-            addText(document, getPage(), pageNo, 500, yOffset, PDType1Font.HELVETICA,12);
+            String pageNo = (pageNumber + 1) + " - " + (pageNumber + noOfPages);
+            if (StringUtils.isNoneBlank(bundle.getPageNumberFormat())
```

Comment:
Rather than using an if statement I think it would be best to have some kind of strategy pattern here so that the enum maps to an implementation in another class